setup.py

```diff
@@ -81,70 +81,74 @@ def copy_data_tree(datadir, destdir):
 gdal2plus = False
 gdal_output = [None] * 4
 gdalversion = None
+gdal_major_version = 0
+gdal_minor_version = 0
+sdist_fill = []
```

Comment:
@sebastic i don't see that these variables are used. Can you delete them?

Reply:
No, they cannot be deleted, because they are used.

Look at the patched file, e.g.:

 https://github.com/mapbox/rasterio/blob/a2a1f5d39112e6fd045a5441ee777ed22adbc515/setup.py#L217
 https://github.com/mapbox/rasterio/blob/a2a1f5d39112e6fd045a5441ee777ed22adbc515/setup.py#L251